### ISSUE TITLE
wrong output compared to running the program

### ISSUE THREAD

**Issue by kostadin-belchev** [wallabyjs/quokka#275]
### Issue description or question
there are less % signs than there should be in the quokka output
running program gives correct result

### Sample code
![image](https://user-images.githubusercontent.com/32366535/47145290-05f97600-d2d2-11e8-83c5-40d650c3e197.png)
```javascript
const progressBar = (progress) => {
  let numOfPercentages = progress / 10
  const PERCENTAGE_SIGN = '%'
  const DOT_SIGN = '.'

  let numOfDots = 10 - numOfPercentages
  if (progress === 100) {
    return `${progress}% Complete!\n[${PERCENTAGE_SIGN.repeat(numOfPercentages)}${DOT_SIGN.repeat(numOfDots)}]`
  } else {
    return `${progress}% [${PERCENTAGE_SIGN.repeat(numOfPercentages)}${DOT_SIGN.repeat(numOfDots)}]\nStill loading...`
  }
}

console.log(progressBar(30))
console.log(progressBar(50))
console.log(progressBar(100))

// output
/* 
30% [%%%.......]
Still loading...

50% [%%%%%.....]
Still loading...

100% Complete!
[%%%%%%%%%%]
*/
```

### Sample repository link

*If the issue can not be reproduced just using the quokka file above 
(for example because it requires/imports some files from your project), 
please create a small repository where the issue can be reproduced.* 

### Quokka.js Console Output

```
​​​​​30% [%%.......]​​​​​
​​​​​Still loading...​​​​​
  at ​​​progressBar(30)​​​ ​04_Funtions and Forms - Exercises/09. Loading Bar.js:14:0​
​​​​​​​​​​
​​​​​50% [%%%.....]​​​​​
​​​​​Still loading...​​​​​
  at ​​​progressBar(50)​​​ ​04_Funtions and Forms - Exercises/09. Loading Bar.js:15:0​
​​​​​​​​​​
​​​​​100% Complete!​​​​​
​​​​​[%%%%%]​​​​​
  at ​​​progressBar(100)​​​ ​04_Funtions and Forms - Exercises/09. Loading Bar.js:16:0​
```

### Code editor version

Visual Studio Code
Version: 1.28.1 (user setup)
Date: 2018-10-11T18:13:53.910Z
Electron: 2.0.9
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64

### OS name and version

Windows 10 Pro

**Comment by ArtemGovorov** [comment]
Thanks for the report. The issue is fixed and the fix will be published with the next version.I'll post some update here when the new version is available.

**Comment by ArtemGovorov** [comment]
The fix is published in v1.0.156.
